testutil/simapp/simapp.go

```diff
@@ -16,35 +15,27 @@ import (
 )
 
 // New creates application instance with in-memory database and disabled logging.
```

Comment:
So when I do these, I generally model after gaia though I'd like to give Pylons the osmosis makeover like we did with craft.  I think it makes the most sense by doing gaia-way for now, and progressing towards osmosis-way.

Reply:
ahhhhh!

OK I get it-- so, check this out:

https://github.com/CosmosContracts/juno/pull/207

we shouldn't need this folder any longer.

When we lose the ignite cli stuff, we no longer need this faux simapp thing.  No wonder why I couldn't map it to gaia or osmo styles :)